app/src/main/java/campus/tech/kakao/map/SearchRepository.kt

```diff
@@ -2,22 +2,35 @@ package campus.tech.kakao.map
 
 import android.content.ContentValues
 import android.content.Context
+import android.provider.BaseColumns
 
 class SearchRepository(context: Context) {
 
     private val dbHelper = StoreInfoDBHelper(context)
 
     fun setDefaultData() {
-        delAllStoreInfo()
+        if(!hasStoreInfo()) return
```

Comment:
```suggestion
        if(hasStoreInfo()) return
```
요구사항대로 동작하려면 코드가 위와 같이 변경되어야 할 것 같아요. ㅎㅎ